src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -502,8 +502,7 @@ def current_value_for(attribute)
           when :mount_point
             filesystem.mount_path
           when :label
-            # Copying the label from the filesystem in the disk looks unexpected
-            new?(filesystem) ? filesystem.label : nil
+            filesystem.label
```

Comment:
Please, reference the bug report here. We re-use the label from the previous file-system but not other attributes. That's surprising to me, so I would tend to consider it a bug and "fix" it, unless it's stated to be intentional.